internal/cri/server/sandbox_portforward_windows.go

```diff
@@ -44,20 +42,17 @@ func (c *criService) portForward(ctx context.Context, id string, port int32, str
 	} else {
 		// HPCs use the host networking namespace.
 		// Therefore, dial to localhost.
-		podIP = "127.0.0.1"
+		podIP = "localhost"
 	}
 
 	err = func() error {
 		var conn net.Conn
-		if netutils.IsIPv4String(podIP) {
-			conn, err = net.Dial("tcp4", fmt.Sprintf("%s:%d", podIP, port))
-			if err != nil {
-				return fmt.Errorf("failed to connect to %s:%d for pod %q: %v", podIP, port, id, err)
-			}
-		} else {
-			conn, err = net.Dial("tcp6", fmt.Sprintf("%s:%d", podIP, port))
+		conn, err = net.Dial("tcp4", fmt.Sprintf("%s:%d", podIP, port))
```

Comment:
Should this just be `tcp` and let the dialer figure out the right one?

Also please use `net.JoinHostPort`.

Reply:
@cpuguy83  I was aware tcp would figure out the right one. Saw this https://pkg.go.dev/net#ListenConfig. We could change to tcp instead. cc @MikeZappa87 

Will update to net.JoinHostPort()